### DISCUSSION TITLE
Piggy Bank - Rule

### DISCUSSION THREAD

**Discussion by bartekd123** [firefly-iii/firefly-iii#4387] [General]
I have created a piggy bank that I have in my savings account for my bills. When it comes time to pay the bill, I transfer the money from my savings acct into my checking acct, and then pay the bill from my checking account.

I have a rule that I setup that takes the money out of my piggy bank when the bill is paid from my checking.

It doesn't seem to be working, and I think I know why. Will the rule work for taking the money out of my piggy bank if the account for the transaction is different than the account that is used for the piggy bank? Is there any way around that, that anyone knows? Or is there something else I can check to see why it is failing?

I upgraded to 5.5.0 beta because of the bug James fixed for rules with piggy bank.

**Comment by JC5** [comment]
Piggy banks can only be influenced by transfers. So the rule should trigger when you move money from the piggy bank, not when you pay the bill.

**Reply by bartekd123** [reply]
thanks James. Was looking through the documentation and through Github, and couldn't find any info on that, so decided to ask here. Thanks for your reply!

**Reply by JC5** [reply]
No problem! I updated the docs as well.
